source/Expression/REPL.cpp

```diff
@@ -295,6 +295,12 @@ void REPL::IOHandlerInputComplete(IOHandler &io_handler, std::string &code) {
       expr_options.SetColorizeErrors(colorize_err);
       expr_options.SetPoundLine(m_repl_source_path.c_str(),
                                 m_code.GetSize() + 1);
+
+      // We don't want other threads to interfere and block the thread
```

Comment:
That comment isn't quite right.  Maybe something like:

// There is no point in trying to run REPL expressions on just the current thread of the program, since the REPL
// track the states of individual threads as you would in a regular debugging session.  Interrupting the
// process to switch from one thread to many has observable effects (for instance it causes anything waiting in
// the kernel to be interrupted).  Since we aren't getting any benefit from doing this in the REPL, let's not.